src/lib/dialog/dialog.ts

```diff
@@ -119,12 +117,26 @@ export class MdDialog {
    */
   private _getOverlayState(dialogConfig: MdDialogConfig): OverlayState {
     let state = new OverlayState();
+    let strategy = this._overlay.position().global();
+    let position = dialogConfig.position;
 
     state.hasBackdrop = true;
-    state.positionStrategy = this._overlay.position()
-        .global()
-        .centerHorizontally()
-        .centerVertically();
+    state.positionStrategy = strategy;
+
+    // TODO(crisbeto): replace with `horizontal` to support RTL?
```

Comment:
I think that if someone is manually setting the position of the dialog that they should be in change of dealing with RTL.